hack/ensure-acr-cleanup-schedule.sh

```diff
@@ -18,10 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-e2e_registry="ci-e2e/cluster-api-azure-controller-amd64"
-az acr task create --name midnight_capz_ci_e2e_purge --cmd "acr purge --filter ${e2e_registry}:.* --ago 1d --untagged" \
-  --schedule "0 0 * * *" --registry capzci --context /dev/null
-
 registry="cluster-api-azure-controller-amd64"
 az acr task create --name midnight_capz_purge --cmd "acr purge --filter ${registry}:.* --ago 1d --untagged" \
   --schedule "0 0 * * *" --registry capzci --context /dev/null
+
+k8s_ci_registry="prow"
+az acr task create --name midnight_capz_purge --cmd "acr purge --filter ${k8s_ci_registry}:.* --ago 1d --untagged" \
```

Comment:
I think this will only purge `prow` but not `prow/.*:.*`

Reply:
testing

Reply:
yeah you're right this isn't going to work... the "repository" gets created as `prow/cluster-api-azure-controller-amd64` and running this cleanup task fails with:

`Deleting manifests for repository: prow
prow repository not found`

Looks like we can do regex on the image tags but not on the image name

/hold

Reply:
```
Deleting tags for repository: prow/*
prow/* repository not found
Deleting manifests for repository: prow/*
prow/* repository not found
```

confirmed